apps/bfd-pipeline/bfd-pipeline-rif-load/src/main/java/gov/cms/bfd/pipeline/rif/load/RifLoader.java

```diff
@@ -705,6 +774,50 @@ static String computeHicnHash(
     }
   }
 
+  /**
+   * Computes a one-way cryptographic hash of the specified MBI value.
+   *
+   * @param options the {@link LoadAppOptions} to use
+   * @param secretKeyFactory the {@link SecretKeyFactory} to use
+   * @param mbi the Medicare beneficiary id to be hashed
+   * @return a one-way cryptographic hash of the specified MBI value, exactly 64 characters long
+   */
+  static String computeMbiHash(
```

Comment:
change this to `computeHash` or `computeIdentifierHash` and use for both hicn's and mbi's.

Note: keeping `computeMbiHash` and `computeHicnHash` and having those functions both call the new one would make sense.

Reply:
Great suggestion! The methods share logic now.